core/src/main/java/io/confluent/kafka/schemaregistry/rest/resources/SubjectVersionsResource.java

```diff
@@ -35,6 +37,7 @@
 import io.confluent.kafka.schemaregistry.exceptions.UnknownLeaderException;
 import io.confluent.kafka.schemaregistry.rest.VersionId;
 import io.confluent.kafka.schemaregistry.rest.exceptions.Errors;
+import io.confluent.kafka.schemaregistry.rest.exceptions.RestInvalidSubjectException;
```

Comment:
nit: is this extraneous?

Reply:
Removed this, btw Should I merge this to master, or should do I pint merge to previous branches? Thanks.